templates/base.html

```diff
@@ -39,18 +42,18 @@
         <nav class="navbar" role="navigation" aria-label="main navigation">
           <div class="navbar-brand" id="brand">
             <a class="navbar-item" href="{{ url_for('home') }}">
-              &nbsp;&nbsp;&nbsp;&#127908;
+              <img  src="{{  url_for('static', filename='images/logo-brand.png') }}">
             </a>
 
             <a class="navbar-item" href="{{ url_for('home') }}">
-              Home
+              Now Playing
             </a>
 
             <a class="navbar-item" href="{{ url_for('queue') }}">
               Queue
             </a>
             <a class="navbar-item" href="{{ url_for('search') }}">
-              Songs
+              Search Songs
             </a>
```

Comment:
The titles were deliberately small to fit on iPhone 5 resolutions, otherwise they start to line break. 

Also, please do not change the logo to a personal logo in a master pull request (line 45)

Reply:
apologies I'll just have to revert is back.